code/game/gamemodes/heist/tools.dm

```diff
@@ -32,15 +32,16 @@
 	)
 	var/target_type="eyes"
 
-/obj/item/weapon/organ_remover/examine(var/mob/user)
-	// Only vox know what these are.
+/obj/item/weapon/organ_remover/examine(var/mob/user, var/override = FALSE)
 	if(ishuman(user))
 		var/mob/living/carbon/human/H
 		if(isvox(H))
 			to_chat(user, "A sophisticated device used by vox raiding parties to remove organs without time-consuming surgical procedures.")
 			to_chat(user, "Subject must be still and incapacitated. Remember to set target organs before use!")
 			return
 	to_chat(user, "Some weird alien thing, doesn't look like it'd even fit in human hands.")
+	if(override)
+		return ..()
```

Comment:
I suggest you put these two lines at the beginning of the proc so you don't get both descriptions:

>Some weird alien thing, doesn't look like it'd even fit in human hands.
>This is an organics extractor. It is a tiny item.
>A knock-off of the vox-only organ extractor, this one has been modified to be able to be used by anyone, and works twice as fast as the real deal. However, it can no longer extract hearts.

Reply:
Already tested it before I went ahead and it only showed the desc

Reply:
Turns out I was wrong about that. But there's still something off here.
![a 1 item](https://user-images.githubusercontent.com/31839805/43370084-751227a6-932d-11e8-8c22-52d6530fea10.PNG)

Reply:
Change the ``..()`` to ``..(user)`` so you're not passing that second argument and making it say 1.

Reply:
Oh.